connectors/redis/redis_test.go

```diff
@@ -232,9 +215,134 @@ func TestEntityTypeNotBytes(t *testing.T) {
 }
 
 func TestShutdownConnector(t *testing.T) {
-	var rc = redis.NewConnector(testRedisConfig)
+	var rc = redis.NewConnector(testRedisConfig, nil)
 	err := rc.Shutdown()
 	assert.NoError(t, err)
 	err = rc.Shutdown()
 	assert.NoError(t, err)
 }
+
+// Test that a successful read from redis logs as cache hit
+func TestReadHitStat(t *testing.T) {
+	if !redis.IsRunning() {
```

Comment:
This tests seem to have some significant copy pasta. Consider refactoring, perhaps pulling out the common stuff into a single function. You also _might_ be able to write a single test that has a for loop in it that just iterates over each of the different permutations you want to test. Like: https://github.com/uber-go/dosa/blob/master/connectors/cassandra/schema_test.go#L95